storage/database/src/main/java/de/danoeh/antennapod/storage/database/DBWriter.java

```diff
@@ -521,6 +522,62 @@ private static void removeQueueItemSynchronous(final Context context,
         }
     }
 
+    public static Future<?> moveQueueItemsToTop(final Context context,
+                                            final List<FeedItem> items) {
+        return runOnDbThread(() -> moveQueueItemsSynchronous(context, true, items));
+    }
+
+    public static Future<?> moveQueueItemsToBottom(final Context context,
+                                                   final List<FeedItem> items) {
+        return runOnDbThread(() -> moveQueueItemsSynchronous(context, false, items));
+    }
+
+    private static void moveQueueItemsSynchronous(final Context context,
+                                                  final boolean moveToTop,
+                                                  final List<FeedItem> items) {
+
+        String calledBy = moveToTop ? "moveQueueItemsToTop" : "moveQueueItemsToBottom";
```

Comment:
I would remove that from the logging calls and just print the `moveToTop` parameter. That reduces complexity for future readers of the code.